service/index.js

```diff
@@ -115,18 +115,26 @@ module.exports.addMessage = (data) => {
 };
 
 module.exports.deletePhoto = (data) => {
-  console.log('delete is here', data.messageId);
   const query = 'DELETE FROM conversation WHERE messageId=$1;';
   const values = [data.messageId];
   return pool.query(query, values)
     .then((response) => {
-      console.log(response);
       if (response.rowCount === 0) {
         return 'noID';
       }
-      const query2 = 'SELECT * FROM conversation WHERE chatId=$1;';
-      const values2 = [data.chatId];
-      return pool.query(query2, values2);
+      return response;
+    })
+    .catch((err) => {
+      console.log(err);
+    });
+};
+
+module.exports.getAfterDelete = (chatId) => {
```

Comment:
This might be open to SQL injection, depending on how the chatId is gotten. Id be happy to meet and talk about it quickly!